dhcp: define $SAVEDIR in dhclient-script

the difination is used by nis and ntp.

Signed-off-by: Li Wang <li.wang@windriver.com>
---
 client/scripts/linux |    1 +
 1 file changed, 1 insertion(+)

diff --git a/client/scripts/linux b/client/scripts/linux
index bbcf354..69619f5 100755
--- a/client/scripts/linux
+++ b/client/scripts/linux
@@ -25,6 +25,7 @@
 # 'ip' just looks too weird.  /sbin/ip looks less weird.
 ip=/sbin/ip
 ETCDIR="/etc/dhcp"
+SAVEDIR=/var/lib/dhcp
 
 make_resolv_conf() {
   if [ x"$new_domain_name_servers" != x ]; then
-- 
1.7.9.5

